Symfony/CS/Tests/Fixer/Symfony/NoEmptyLinesAfterPhpdocsFixerTest.php

```diff
@@ -212,4 +212,26 @@ public function testFixesWindowsStyle()
 
         $this->makeTest($expected, $input);
     }
+
+    // empty line between typehinting docs and return statement should be preserved
+    public function testInlineTypehintingDocsBeforeReturn()
+    {
+        $expected = <<<'EOF'
+<?php
+
+function parseTag($tag)
+{
+    $tagClass = get_class($tag);
+
+    if ('phpDocumentor\Reflection\DocBlock\Tag\VarTag' === $tagClass) {
+        /** @var DocBlock\Tag\VarTag $tag */
```

Comment:
The return in that if uses a `$tag`, which is not typehinted by code and therefore is typehinted by inline docblock.

We met a lot of odd cases when autofixing, didn't we? Here is one of them - found on my pre-regression of v1.5.

It's weird, you won't use it, but one still could, and the line should be preserved.